packages/contracts-bedrock/test/L2/L2StandardBridge.t.sol

```diff
@@ -14,18 +14,50 @@ import { Hashing } from "src/libraries/Hashing.sol";
 import { Types } from "src/libraries/Types.sol";
 
 // Target contract dependencies
+import { L2StandardBridge } from "src/L2/L2StandardBridge.sol";
 import { Predeploys } from "src/libraries/Predeploys.sol";
 import { StandardBridge } from "src/universal/StandardBridge.sol";
 import { OptimismMintableERC20 } from "src/universal/OptimismMintableERC20.sol";
 
 contract L2StandardBridge_Test is Bridge_Initializer {
     using stdStorage for StdStorage;
 
+    /// @dev Test that the bridge's constructor sets the correct values.
+    function test_constructor_succeeds() external {
+        L2StandardBridge impl = L2StandardBridge(deploy.mustGetAddress("L2StandardBridge"));
+        assertEq(address(impl.MESSENGER()), address(l2CrossDomainMessenger));
+        assertEq(address(impl.messenger()), address(l2CrossDomainMessenger));
+        assertEq(l1StandardBridge.l2TokenBridge(), address(impl));
+        assertEq(address(impl.OTHER_BRIDGE()), address(l1StandardBridge));
+        assertEq(address(impl.otherBridge()), address(l1StandardBridge));
+    }
+
     /// @dev Tests that the bridge is initialized correctly.
     function test_initialize_succeeds() external {
+        assertEq(address(l2StandardBridge.MESSENGER()), address(l2CrossDomainMessenger));
         assertEq(address(l2StandardBridge.messenger()), address(l2CrossDomainMessenger));
         assertEq(l1StandardBridge.l2TokenBridge(), address(l2StandardBridge));
         assertEq(address(l2StandardBridge.OTHER_BRIDGE()), address(l1StandardBridge));
+        assertEq(address(l2StandardBridge.otherBridge()), address(l1StandardBridge));
+    }
+
+    /// @dev Tests that the implementation contract cannot be initialized twice.
+    function test_initializeImpl_alreadyInitialized_reverts() external {
```

Comment:
Can you double check that `Initializable.t.sol` checks both the proxy and the impl? If so then these can be deleted

Reply:
in Initializable.t.sol, it looks like proxy and impl are being checked but for different chains (i.e., proxy for L1ERC721Bridge test, impl for L2ERC721Bridge). wouldn't you be missing checking impl of L1ERC721Bridge or proxy for L2ERC721Bridge?

Reply:
We spoke about breaking out an individual PR that updates the initializable test on its own

```
step 1: rename the names so that the proxy suffix is included when proxy: true
step 2: update proxy: true functionality such that when true it looks up impl slot addr in the contract address corresponding to name
then for predeploys, there is one test where proxy:true and one test where proxy:false
maybe change proxy to proxied
so it reads more clearly
```